src/Components/Artsy.tsx

```diff
@@ -1,14 +1,16 @@
 import PropTypes from "prop-types"
 import React from "react"
 
+// TODO: Once this PR for `rest` https://github.com/Microsoft/TypeScript/pull/13470 lands
+// we’ll be able to not make this optional and simply remove it from the props that a
+// component wrapped with the `ContextConsumer` HOC accepts.
+
```

Comment:
@eessex Curious what the reason was for this, was it cluttering up the doc view in VSCode?

Reply:
The todo was causing a type error I believe- but my memory is a bit foggy now

Reply:
Oh weird. Ok, no problemo, just wondering 👍